proto/akash/escrow/v1beta1/types.proto

```diff
@@ -53,19 +53,31 @@ message Account {
     (gogoproto.moretags) = "yaml:\"state\""
   ];
 
-  cosmos.base.v1beta1.Coin balance   = 4 [
+  cosmos.base.v1beta1.Coin credits_balance   = 4 [
     (gogoproto.nullable) = false,
     (gogoproto.jsontag)  = "balance",
-    (gogoproto.moretags) = "yaml:\"balance\""
+    (gogoproto.moretags) = "yaml:\"creditsBalance\""
+  ];
+
+  cosmos.base.v1beta1.Coin tokens_balance   = 5 [
+    (gogoproto.nullable) = false,
+    (gogoproto.jsontag)  = "tokensBalances",
+    (gogoproto.moretags) = "yaml:\"tokensBalance\""
+  ];
+
+  cosmos.base.v1beta1.Coin total_balance   = 6 [
+    (gogoproto.nullable) = false,
+    (gogoproto.jsontag)  = "totalBalance",
+    (gogoproto.moretags) = "yaml:\"totalBalance\""
   ];
 
-  cosmos.base.v1beta1.Coin transferred = 5 [
+  cosmos.base.v1beta1.Coin transferred = 7 [
```

Comment:
We shouldn't change the tag numbers, newly added tags should have new tag numbers